flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/ZooKeeperHADispatcherTest.java

```diff
@@ -167,7 +181,7 @@ public void testSubmittedJobGraphRelease() throws Exception {
 				// recover the job
 				final SubmittedJobGraph submittedJobGraph = otherSubmittedJobGraphStore.recoverJobGraph(jobId);
 
-				assertThat(submittedJobGraph, Matchers.is(Matchers.notNullValue()));
+				assertThat(submittedJobGraph, is(Matchers.notNullValue()));
```

Comment:
You added a static import for `is` but not for `notNullValue`. I think this should be consistent.

Reply:
Good catch. Will change it.